packages/genesys-spark-components/src/utils/dom/wrap-text-nodes.ts

```diff
@@ -0,0 +1,12 @@
+export function wrapTextNodes(element: HTMLElement): void {
```

Comment:
Want to know peoples thoughts on creating a helper function which will wrap text nodes in a span. I am currently using this in `gux-form-heading` with this function you will be able to pass both elements and text nodes at the same time and the styling will apply.

Reply:
This creates the span in the light DOM, right? If so, I don't think we can support this, as our changes will be in conflict with what the application is trying to render. I think that's one of our rules that we can't make exceptions to - the component's aren't allowed to change the structure of the light DOM, as it's owned by the hosting app/page.

Reply:
Thats true I was thinking there will be caveats with this solution but trying to avoid the named slot route. Maybe logging a warning is another option but will continue to look. As of now there is now selector to style text nodes.